packages/component-library/src/utils/formatters.test.js

```diff
@@ -0,0 +1,20 @@
+import { numeric, percentage, dollars } from './formatters';
+
+describe('formatters', () => {
+  it('should format numbers correctly', () => {
+    expect(numeric(0.1)).to.eql('0');
+    expect(numeric(0.5)).to.eql('1');
+    expect(numeric(1000)).to.eql('1,000');
+    expect(numeric(1000000)).to.eql('1 million');
+    expect(numeric(7300000)).to.eql('7.3 million');
+  });
+
+  it('should format percentages correctly', () => {
+    expect(percentage(0.75)).to.eql('75%');
+    expect(percentage(0.2379)).to.eql('24%');
+  });
+
+  it('should format dollars correctly', () => {
+    expect(dollars(12)).to.eql('$12');
```

Comment:
Could be nice to write tests for larger dollars, fractions of dollars, and dollars working with numeric. IMO tests are a great way to also document usage.